### ISSUE TITLE
[Bug] not able to add to home assistant add on store "Not a valid add-on repository"

### ISSUE THREAD

**Issue by pbanj** [JoshuaMulliken/ha-wyzeapi#89]
the error
![image](https://user-images.githubusercontent.com/17306233/95793906-a95ca600-0cb4-11eb-8822-c91b86ebed0c.png)

my home assistant info
![image](https://user-images.githubusercontent.com/17306233/95793966-cc875580-0cb4-11eb-9fe1-e6056b24285b.png)

**Comment by JoshuaMulliken** [comment]
I need more details. Are you trying to add this to HACS?

**Comment by oliviercar0n** [comment]
same issue here

**Comment by JoshuaMulliken** [comment]
This integration is not an add on and will not work in this way. Please read the readme for instructions on adding the repo to the Home Assistant Community Store

**Comment by LordNex** [comment]
I was able to add it to HACS but the install button is grayed out. I checked my logs but don't see anything at the moment related to this plugin. 

For the record, I have 4 pan cams and 1 Outdoor can with base station. I have a Wyze lock coming next week and am really looking forward to getting it integrated into HA. 

I have some other questions regarding integration and implementation but I'll save those for when I get the lock and another thread.
